pages/signup/index.tsx

```diff
@@ -58,26 +61,26 @@ const SignUpPage = ({ t }: PageProps): React.ReactElement => {
       />
     ) : null;
 
-  const handleSubmit = async (
-    e: React.FormEvent<HTMLFormElement>,
-  ): Promise<void> => {
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
     e.preventDefault();
-    // const newUser = {
-    //   name: formValues.name,
-    //   email: formValues.email,
-    //   password: formValues.password,
-    //   type: formValues.type,
-    // };
-    // if (formValues.company) newUser.company = formValues.company;
-    console.log(formValues);
-    try {
-      await createUser({
-        variables: {
-          input: formValues,
-        },
-      });
-    } catch (e) {
-      console.error('user already exists: ', e.message);
+    if (formValues.email && formValues.password) {
```

Comment:
is there a reason that we're doing then rather than async await? Don't mind, just curious.

Reply:
I think I used Promises more because the Firebase Docs use Promises as well